QUEST_GUIDELINES.md

```diff
@@ -19,7 +19,7 @@ Consider the following:
 - 🤔 **No unanswerable quests**: All generated quests need to be actually answerable (no false-positives). This means that any answer given by the user must result in something being tagged. For example, a quest that asks for the website of a place must be able to tag the element somehow if the user answers that the place has no website - otherwise, the next user will be asked the same question. Sometimes, due to the nature of how things are tagged in OSM (such as the one given in the example) it is unfortunately simply not possible to fulfill this.
 - 👨‍💻 **Users are no experts**: No knowledge about OpenStreetMap or any other background knowledge must be necessary
 - 🐿️ **Easy answer**: Users are out and about and impatient. A quick, straightforward and clear answer must be possible
-- 🕵️ **Publicly accessible**: The information the users are supposed to collect must be easily accessible from the outside, don't lead users into trouble. I.e. users will not be able to answer reliably if a building has a basement or not, if a restaurant has toilets or not etc. without entering it.
+- 🕵️ **Publicly accessible on foot**: The information the users are supposed to collect must be easily accessible from the outside by pedestrians, don't lead users into trouble. I.e. users will not be able to answer reliably if a building has a basement or not, if a restaurant has toilets or not etc. without entering it. Also information should not require special means of transport like a car or a bus (e.g. motorway surveying).
```

Comment:
Erm we specifically ask a number of those example quests which are deemed unsuitable (and other equivalents), although I appreciate they may be mostly disabled by default, shouldn't we call that out here?